src/ucp/core/ucp_worker.c

```diff
@@ -2133,9 +2158,10 @@ void ucp_worker_destroy(ucp_worker_h worker)
     UCS_STATS_NODE_FREE(worker->stats);
     ucs_ptr_map_destroy(&worker->ptr_map);
     ucs_strided_alloc_cleanup(&worker->ep_alloc);
+    kh_destroy_inplace(ucp_worker_rkey_config, &worker->rkey_config_hash);
     kh_destroy_inplace(ucp_worker_discard_uct_ep_hash,
                        &worker->discard_uct_ep_hash);
-    kh_destroy_inplace(ucp_worker_rkey_config, &worker->rkey_config_hash);
+
```

Comment:
no need to move
must keep destroy order in sync with init order

Reply:
fixed